main.py

```diff
@@ -0,0 +1,74 @@
+import re
+import datetime
+from collections import OrderedDict
+
+def create_racer_abbreviations_dict(file_name):
+    """Retrieves {'abbreviation': (name, team)}" format dict from abbreviations.txt"""
+    abbreviations_dict = {}
+    with open(file_name, 'r') as fn:
+        for line in fn:
+            matchObj = re.match(r'^(\w+)_([a-zA-Z\s]+)_([a-zA-Z\s]+)$', line)
+            # group(1) is abbreviation, i.e 'SVM'
```

Comment:
I would suggest you use variables instead of docstrings to make your code explain itself
So instead you can write 
```
abbreviation = group(1)
pilot_name = group(2)
team_name = group(3)
abbreviations_dict[abbreviation] = (pilot_name, team_name)
```

it is more rows but easier to understand